src/loading.js

```diff
@@ -0,0 +1,18 @@
+import '../style/preview.less'
+
+const renderLoading = ( isTouch ) => {
```

Comment:
Actually thinking thing of moving `renderLoading` to preview.js, anyone agree that's better?

Reply:
we don't have one general loading behaviour, instead we have the different loading in preview / gallery.

therefore I think we can have the `previewLoading` component, then `preview` component use it.
same to gallery popup loading component.

Reply:
Yea looking at this again, `renderLoading` seems sort of lonely in loading.js by itself. I think it makes sense to have it make some company to `renderPreview` in preview.js, they are are both aligned shared some logic